### DISCUSSION TITLE
What's the best way to clamp camera rotation vertical angle

### DISCUSSION THREAD

**Discussion by epcau** [Sixze/ALS-Refactored#386] [Q&A] [answered]
I would like to reduce the vertical range / angle the camera can rotate around the character, I feel like a range is already defined somewhere but I can't find it

**Comment by ameaninglessname** [comment, marked as answer]
Looks like you want to use `APlayerCameraManager::LimitViewPitch`.

**Reply by epcau** [reply]
Ah nice! Didn't even know about the camera manager class, thanks a lot!
